androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/TextArea.kt

```diff
@@ -33,8 +33,6 @@ fun TextAreaPreview(
     @PreviewParameter(SurveyDetailScreenPreviewParameterProvider::class) params: SurveyDetailScreenPreviewParameterProvider.Params,
 ) {
     ApplicationTheme {
-        TextArea(
-            surveyAnswerUiModel = params.survey.questionUiModels[0].answers[0]
-        )
+        TextArea(surveyAnswerUiModel = answerUiModels[0])
```

Comment:
This also needs to show only one preview. And please correct me if I am wrong, the branch that implemented this feature was not merged into the develop. It was just closed. How did we carry the feature to this stage? 💭

Reply:
PR #123 has been reopened. Please review that.